stl/src/fiopen.cpp

```diff
@@ -81,12 +78,12 @@ FILE* _Xfiopen(const CharT* filename, ios_base::openmode mode, int prot) {
         return nullptr; // open failed

```

Comment:
`n` on the line above used to denote the index determined by the loop; I'm not sure what (if anything) it now names.

Reply:
It still does now.